cognite/client/_http_client.py

```diff
@@ -107,24 +107,28 @@ def __init__(
         self.refresh_auth_header = refresh_auth_header
         self.retry_tracker_factory = retry_tracker_factory  # needed for tests
 
-    def request(self, method: str, url: str, **kwargs: Any) -> requests.Response:
+    def request(self, method: str, url: str, accept: str, **kwargs: Any) -> requests.Response:
         retry_tracker = self.retry_tracker_factory(self.config)
         headers = kwargs.get("headers")
+        accepts_json = accept == "application/json"
+        is_auto_retryable = False
```

Comment:
Should this be inside the `while` loop?

Reply:
It can be outside, `is_auto_retryable` is always static for a given endpoint.